ocs_ci/ocs/resources/pvc.py

```diff
@@ -120,3 +123,34 @@ def get_all_pvcs(namespace=None):
     )
     out = ocp_pvc_obj.get()
     return out
+
+
+def verify_pv_exists_in_backend(
+        pv_name, pool_name, namespace=defaults.ROOK_CLUSTER_NAMESPACE
+):
+    """
+    Verifies given pv exists in ceph backend
+
+    Args:
+        pvc_name (str): Name of the pvc
+    Returns:
+         bool: True if pv exists on backend, False otherwise
+    """
+    spec_volhandle = "'{.spec.csi.volumeHandle}'"
+    cmd = f"oc get pv/{pv_name} -o jsonpath={spec_volhandle} -n {namespace}"
+    out = run_cmd(cmd=cmd)
+    image_uuid = "-"
+    image_uuid = image_uuid.join(out.split('-')[5:10])
+    cmd = f"rbd info -p {pool_name} csi-vol-{image_uuid}"
+    ct_pod = pod.get_ceph_tools_pod()
+    try:
+        pv_info = ct_pod.exec_ceph_cmd(
+            ceph_cmd=cmd, format='json'
+        )
+    except CommandFailed as ecf:
+        assert f"Error is rbd: error opening image csi-vol-{image_uuid}" in str(ecf), (
```

Comment:
This function should return a bool.
Log the issue and return False

Reply:
removed and adding log.error